src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -130,13 +137,26 @@ private static async Task<string> GetTestData(string testDataPath, ScriptJobHost
         private static Uri GetFunctionHref(string functionName, string baseUrl) =>
             new Uri($"{baseUrl}/admin/functions/{functionName}");
 
-        private static Uri GetFunctionInvokeUrlTemplate(string baseUrl, string functionName, IWebJobsRouter router)
+        private static Uri GetFunctionInvokeUrlTemplate(string baseUrl, FunctionMetadata functionMetadata, string routePrefix)
         {
-            var template = router?.GetFunctionRouteTemplate(functionName);
-
-            if (template != null)
+            var httpBinding = functionMetadata.InputBindings.FirstOrDefault(p => string.Compare(p.Type, "httpTrigger", StringComparison.OrdinalIgnoreCase) == 0);
+            JToken value = null;
+            string template = null;
+            if (httpBinding != null)
             {
-                return new Uri($"{baseUrl}/{template}");
+                if (httpBinding.Raw != null && httpBinding.Raw.TryGetValue("route", StringComparison.OrdinalIgnoreCase, out value))
```

Comment:
Can we inline the `value` declaration here?

Reply:
done